clients/csharp-uwp/UWPVoiceAssistantSample/MainPage.xaml.cs

```diff
@@ -101,20 +98,20 @@ private void AddUIHandlersAsync()
             {
                 WindowService.CloseWindow();
             };
-            this.StartListeningButton.Click += async (_, __) =>
+            this.MicrophoneButton.Click += async (_, __) =>
             {
                 this.dialogManager.HandleSignalDetection(DetectionOrigin.FromPushToTalk);
                 await this.UpdateUIForSharedStateAsync();
             };
-            this.StopListeningButton.Click += async (_, __) =>
+            this.ResetButton.Click += async (_, __) =>
             {
                 await this.dialogManager.FinishConversationAsync();
             };
-            this.StopPlaybackButton.Click += async (_, __) =>
+            this.ResetButton.Click += async (_, __) =>
```

Comment:
Let's merge these ResetButton.Click handlers into one handler. Just run the async methods as fire-and-forget.

Reply:
Done